lesson1/assignment/yours.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.4.14;
+contract Payroll {
+    uint constant payDuration = 10 seconds;
+
+    address boss = 0xca35b7d915458ef540ade6068dfe2f44e8fa733c;
+    uint salary;
+    address employee;
+    uint lastPayday = now;
+
+    //获取员工地址和工资
+    function updateEmployee(address e,uint s) {
+        employee = e;
+        salary = s * 1 finney;
+    }
+    //存钱
+    function addFund() payable returns (uint) {
+        return this.balance;
+    }
+    //余额是否够发工资
+    function hasEnoughFund() returns (bool) {
+        return this.balance / salary > 1;
```

Comment:
The unit for this.balance is wei while the unit for salary seems to be finney according to line 13. Please unify the units.